### ISSUE TITLE
Namespace issues with latest conda-forge package

### ISSUE THREAD

**Issue by nabobalis** [conda-forge/hypothesis-feedstock#16]
We have updated to the lastest version on conda-forge and it fails to import. 

```python
>>> import hypothesis
>>> dir(hypothesis)
['__doc__', '__loader__', '__name__', '__package__', '__path__', '__spec__']
```

The pip version for comparison 
```python
>>> import hypothesis
>>> dir(hypothesis)
['HealthCheck', 'Phase', 'Verbosity', '__all__', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '__version__', '__version_info__', '_settings', 'assume', 'configuration', 'control', 'core', 'errors', 'event', 'example', 'executors', 'find', 'given', 'infer', 'internal', 'note', 'reject', 'reporting', 'searchstrategy', 'seed', 'settings', 'statistics', 'strategies', 'unlimited', 'utils', 'vendor', 'version']
```

**Comment by rth** [comment]
I can confirm this, also related to [this comment](https://github.com/conda-forge/hypothesis-feedstock/pull/15/files#r146685449)..

**Comment by omad** [comment]
Can also confirm that this is broken.

Most annoying is that it blocks `pytest` from running too, since a pytestplugin is registered and not available.

```
ModuleNotFoundError: No module named 'hypothesis.extra.pytestplugin'
```

**Comment by jochym** [comment]
I guess this was a bad idea to merge it anyway. Me bad. Any ideas what will be the best action now?

**Comment by Cadair** [comment]
I think the simpliest solution will be to increment the build number and push an update as soon as possible.

**Comment by ocefpaf** [comment]
I'm removing the bad package for now. I cannot look into fixing the latest version at the moment but I'll try to fix this next week.

**Comment by jochym** [comment]
Thanks @ocefpaf ! The new version is simply broken. I'll try to figure the solution myself as well. The package is simply convoluted and non-standard. I wonder if we should involve some of the authors into this effort. Sorry for being trigger happy with the merge. I forgot that I had problems with building new version and when I have returned to it I did not check the history and when new release merged into PR without problems I just merged ...
